rllib/agents/ppo/tests/test.py

```diff
@@ -1,43 +0,0 @@
-import unittest
```

Comment:
None of these functions were needed anywhere. Removed helper functions and their tests entirely.